### PR TITLE
Get deployment flow back up and running

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4850/
Test FAILed.

**Comment by hatched** [comment]
#### To QA
Talk to me for qa instructions

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4854/
Test PASSed.

File: jujugui/static/gui/src/app/components/deployment/add-credentials/test-add-credentials.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ describe('DeploymentAddCredentials', function() {
     var renderer = jsTestUtils.shallowRender(
       <juju.components.DeploymentAddCredentials
         changeState={sinon.stub()}
+        controller='my-controller'
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/deployment/add-credentials/test-add-credentials.js]: `"` Double quotes for JSX I think.

File: jujugui/static/gui/src/app/components/deployment/summary/test-summary.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ describe('DeploymentSummary', function() {
         env={{}}
         appSet={sinon.stub()}
         createSocketURL={sinon.stub()}
+        controller='yellow/aws-eu-central'
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/deployment/summary/test-summary.js]: Double quotes in this file too.

File: jujugui/static/gui/src/app/components/deployment/test-deployment.js [hunk 3]
```diff
@@ -75,6 +75,7 @@ describe('Deployment', function() {
         env={env}
         appSet={appSet}
         createSocketURL={createSocketURL}
+        controller='yellow/aws-us-east'
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/deployment/test-deployment.js]: Double quotes

**Comment by huwshimi** [comment]
:+1: with a few trivial things

**Comment by hatched** [comment]
Thanks all! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4858/
Test PASSed.

**Pull request merged**